ovh/resource_dbaas_logs_output_opensearch_alias.go

```diff
@@ -0,0 +1,405 @@
+				Description: "Alias used",
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Operation creation",
+			},
+			"current_size": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "Current alias size (in bytes)",
+			},
+			"is_editable": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Indicates if you are allowed to edit entry",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Alias name",
+			},
+			"nb_index": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Number of index",
+			},
+			"nb_stream": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Number of shard",
+			},
+			"updated_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Operation last update",
+			},
+
+			"indexes": {
+				Type:        schema.TypeList,
+				Description: "Indexes attached to alias",
+				Optional:    true,
+				Computed:    true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"streams": {
+				Type:        schema.TypeList,
+				Description: "Streams attached to alias",
+				Optional:    true,
+				Computed:    true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+		},
+	}
+}
+
+func resourceDbaasLogsOutputOpensearchAliasImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	givenID := d.Id()
+	splitID := strings.SplitN(givenID, "/", 2)
+	if len(splitID) != 2 {
+		return nil, fmt.Errorf("Import Id is not service_name/id formatted")
+	}
+	serviceName := splitID[0]
+	id := splitID[1]
+	d.SetId(id)
+	d.Set("service_name", serviceName)
+
+	results := make([]*schema.ResourceData, 1)
+	results[0] = d
+	return results, nil
+}
+
+func resourceDbaasLogsOutputOpensearchAliasCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*Config)
+
+	serviceName := d.Get("service_name").(string)
+
+	log.Printf("[DEBUG] Will create dbaas logs output opensearch alias for: %s", serviceName)
+
+	opts := (&DbaasLogsOutputOpensearchAliasCreateOps{}).FromResource(d)
+	res := &DbaasLogsOperation{}
+	endpoint := fmt.Sprintf(
+		"/dbaas/logs/%s/output/opensearch/alias",
+		url.PathEscape(serviceName),
+	)
+	if err := config.OVHClient.Post(endpoint, opts, res); err != nil {
+		return diag.Errorf("Error calling post %s:\n\t %q", endpoint, err)
+	}
+
+	// Wait for operation status
+	op, err := waitForDbaasLogsOperation(ctx, config.OVHClient, serviceName, res.OperationId)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	id := op.AliasId
+	if id == nil {
+		return diag.Errorf("Alias Id is nil. This should not happen: operation is %s/%s", serviceName, res.OperationId)
+	}
+
+	d.SetId(*id)
+
+	indexes := d.Get("indexes").([]interface{})
```

Comment:
```suggestion
	indexes := d.Get("indexes").([]string)
```

and envoid the next `index.(string)`?

Reply:
```
panic: interface conversion: interface {} is []interface {}, not []string
```
I suppose this is due to this declaration https://github.com/ovh/terraform-provider-ovh/commit/e7dbabe0cce487816643676baf4b4e8d080c1b0c#diff-c62ab4e0d2d514494a5e0c1f4e955ccdf85a59d2fcd1b2b59ca14f7e32b8e79dR88